lib/livebook_web/live/form_helpers.ex

```diff
@@ -0,0 +1,74 @@
+defmodule LivebookWeb.FormHelpers do
+  @moduledoc """
+  Conveniences for translating and building error messages.
+  """
+
+  use Phoenix.Component
+
+  import Phoenix.HTML.Form
+  import LivebookWeb.LiveHelpers
+
+  @doc """
+  A wrapper for inputs with conveniences.
+  """
+  def input_wrapper(assigns) do
+    assigns = assign_new(assigns, :class, fn -> [] end)
+
+    ~H"""
+    <div
+      phx-feedback-for={input_name(@form, @field)}
+      class={[@class, if(@form.errors[@field], do: "show-errors", else: "")]}
+    >
+      <%= render_slot(@inner_block) %>
+      <%= for error <- Keyword.get_values(@form.errors, @field) do %>
+        <span class="hidden text-red-600 text-sm phx-form-error:block">
+          <%= translate_error(error) %>
+        </span>
+      <% end %>
+    </div>
+    """
+  end
+
+  @doc """
+  Hex color input.
+
+  It expects a phx-click to be assigned to a randomize function.
```

Comment:
What about a more specialized assign like `@on_randomize`?

Reply:
Because we need both phx-click and phx-target, i decided to not abstract it too much for now. Preferences?

Reply:
We can do `on_randomize={JS.push("randomize_color", target: @myself)}` :)